javascript-if/main.js

```diff
@@ -0,0 +1,65 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(num) {
+  if (num < 5) {
+    return true;
+  }
+}
+
+function isEven(num) {
+  if (num % 2 === 0) {
+    return true;
+  }
+}
+
+function startsWithJ(str) {
+  if (str[0] === 'J') {
+    return true;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  }
+}
```

Comment:
Make sure you are returning `false` if the condition is not met. As of right now, you are returning `undefined`, which is also falsy, but not strictly `false`

Reply:
![conditionals2](https://user-images.githubusercontent.com/76453386/111546448-f0a00480-8734-11eb-88b1-220f66af2133.png)